src/test/java/features/Dropdown Tests.feature

```diff
@@ -7,31 +7,31 @@ Feature: Dropdown Tests
   Scenario: 273 Dropdown
     Given I am on the Dropdown Page
     When I select Option 1 from the Dropdown
-      And I verify the Dropdown has the text "Option 1" selected
+      And I verify the Dropdown has the text "Option 1"
```

Comment:
If we added `(?:.*)` to the end of line 151 in TextVerificationSteps, the wording of this step wouldn't have to change and the quotes would make more sense.